src/main/java/me/karakelley/http/controllers/RootController.java

```diff
@@ -0,0 +1,27 @@
+package me.karakelley.http.controllers;
+
+import me.karakelley.http.Request;
+import me.karakelley.http.Response;
+import me.karakelley.http.Status;
+
+public class RootController implements Controller {
+  private final Request request;
+  private final String DEFAULT_RESPONSE = "Hello World";
+
+  public RootController(Request request) {
+    this.request = request;
+  }
+
+  public Response respond() {
+    Response response = new Response();
+    if (request.validRequestLine() && request.getMethod().equals("GET")) {
```

Comment:
Possibly still too early for an abstraction, but this exact logic also exists in the `RedirectController`